### ISSUE TITLE
ctdet inference time

### ISSUE THREAD

**Issue by XiaoJiNu** [xingyizhou/CenterNet#219]
I run dla_34 ctdet model on 1080ti and the inference time as follows that  is about 42ms, but the papaer  says that the speed is about 52fps. I want to know what's the problem?
tot 0.044s |load 0.000s |pre 0.013s |net 0.026s |dec 0.002s |post 0.003s |merge 0.000s |
tot 0.042s |load 0.000s |pre 0.014s |net 0.023s |dec 0.003s |post 0.002s |merge 0.000s |
tot 0.041s |load 0.000s |pre 0.013s |net 0.024s |dec 0.002s |post 0.002s |merge 0.000s |

**Comment by xingyizhou** [comment]
Hi, the problem is most likely in your CPU. The pre-processing time is abnormal. You can have a reference [here](https://github.com/xingyizhou/CenterNet/issues/15). Also note that 52fps in the paper only includes the `net` and `dec`.

**Comment by XiaoJiNu** [comment]
@xingyizhou thank you, the problem may be the pre-processing with opencv

**Comment by varghesealex90** [comment]
@xingyizhou  I see the pre-processing is a bit heavy [ computing affine tranforms, applying affine transforms etc.]. Have you tried with when the pre-processing is just cv2.resize. Is there a drop in performance?

**Comment by XiaoJiNu** [comment]
@varghesealex90 I didn't try it...I don't understand why computing affine tranforms, applying affine transforms

**Comment by twangnh** [comment]
@XiaoJiNu should the bar like 
`Tot: 0:02:10 |ETA: 0:03:01 |post 0.002s (0.003s) |merge 0.000s (0.001s) |load 0.000s (0.000s) |net 0.008s (0.048s) |pre 0.001s (0.001s) |tot 0.012s (0.054s) |dec 0.001s (0.002s)`?
for which out brackets number is current sample time, while in brackets number is average time?

**Comment by XiaoJiNu** [comment]
@twangnh my  output is not as yours,  I run demo.py and output as following
 tot 0.044s |load 0.000s |pre 0.013s |net 0.026s |dec 0.002s |post 0.003s |merge 0.000s |
i don't knowa what do the time in brackets mean...

**Comment by twangnh** [comment]
Thanks for sharing

**Comment by xingyizhou** [comment]
Yes, the time in brackets is the average time. I haven't tried cv2.resize. It will change the aspect ratio of the image. If you don't like affine transform, you can use crop-and-resize in the CornerNet repo.
